include/zephyr/dt-bindings/clock/stm32f7_clock.h

```diff
@@ -82,6 +82,20 @@
 #define MCO1_PRE(val)           STM32_MCO_CFGR(val, 0x7, 24, CFGR_REG)
 #define MCO2_SEL(val)           STM32_MCO_CFGR(val, 0x3, 30, CFGR_REG)
 #define MCO2_PRE(val)           STM32_MCO_CFGR(val, 0x7, 27, CFGR_REG)
+
+/* MCO prescaler : division factor */
+#define MCO1_PRE_DIV_1 0
+#define MCO1_PRE_DIV_2 4
+#define MCO1_PRE_DIV_3 5
+#define MCO1_PRE_DIV_4 6
+#define MCO1_PRE_DIV_5 7
+
+#define MCO2_PRE_DIV_1 0
+#define MCO2_PRE_DIV_2 4
+#define MCO2_PRE_DIV_3 5
+#define MCO2_PRE_DIV_4 6
+#define MCO2_PRE_DIV_5 7
```

Comment:
Since value is always the same between MCO 1 and MCO 2, lets provide only one set of definitions for each series.
```
#define MCO_PRE_DIV_1 0
#define MCO_PRE_DIV_2 4
#define MCO_PRE_DIV_3 5
#define MCO_PRE_DIV_4 6
#define MCO_PRE_DIV_5 7
```